bats/docs.bats

```diff
@@ -220,35 +184,26 @@ teardown() {
     echo readme-text > README.md
     run dolt add .
     [ "$status" -eq 0 ]
-    run dolt commit -m "first docs commit"
+    dolt commit -m "first docs commit"
     [ "$status" -eq 0 ]
-    [[ "$output" =~ "first docs commit" ]]
-    echo 'updated readme' > README.md
-    run dolt status
+    echo updated readme > README.md
+    dolt status
     [ "$status" -eq 0 ]
-    [[ "$output" =~ "Changes not staged for commit:" ]] || false
-    [[ "$output" =~ ([[:space:]]*modified:[[:space:]]*README.md) ]] || false
-    run cat README.md
-    [ "$output" = "updated readme" ]
-    run dolt reset --hard
+    dolt reset --hard
     [ "$status" -eq 0 ]
```

Comment:
I think this will break. $status after no run...or it will just use the $status from the previous run